test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -483,33 +492,56 @@ def verify_option_exists(self, option:str):
             title = get_obj(CommunitySettingsComponents.MEMBERS_BUTTON.value).title
             verify_text(option, str(title))
         elif option=="Mint Tokens":
-            title = get_obj(CommunitySettingsComponents.MEMBERS_BUTTON.value).title
+            title = get_obj(CommunitySettingsComponents.MINT_TOKENS_BUTTON.value).title
             verify_text(option, str(title))         
               
     def select_community_settings_option(self, option:str):
         if option=="Permissions":
             click_obj_by_name(CommunitySettingsComponents.PERMISSIONS_BUTTON.value)
+        elif option=="Mint Tokens":
+            click_obj_by_name(CommunitySettingsComponents.MINT_TOKENS_BUTTON.value)
      
-    def verify_permission_screen_title(self, option:str):
+    def verify_welcome_screen_title(self, option:str):
```

Comment:
This method should be removed since we already have `verify_welcome_title` that does the same job!